### PR TITLE
EC-971 Reflow - low res drawer accessibility issues

### PR DESCRIPTION
**Pull request by johnholland-nice**

### CODE REVIEW

File: Comments/ClientApp/src/components/CommentList/CommentList.js [hunk 1]
```diff
@@ -284,8 +284,8 @@ export class CommentList extends Component<PropsType, StateType> {
 
 	drawerClassnames = () => {
 		const open = this.state.drawerOpen ? "Drawer--open" : "";
-		const mobile = this.state.drawerMobile ? "Drawer--mobile" : "";
-		return `Drawer ${open} ${mobile}`;
+		//const mobile = this.state.drawerMobile ? "Drawer--mobile" : "";
```
Comment (emollett) [inline Comments/ClientApp/src/components/CommentList/CommentList.js:287]: Can this be removed?

File: Comments/ClientApp/src/components/CommentList/Drawer.scss [hunk 2]
```diff
@@ -12,24 +12,21 @@
...
-    }
+    transform: translateX(-75px);
   }
 
   &__control {
+    font-size: 0;
     border-radius: rem(10px) 0 0 rem(10px);
     line-height: 1.3;
-    padding: rem($spacing-medium);
+    padding: 0.5rem 0.3rem;
```
Comment (emollett) [inline Comments/ClientApp/src/components/CommentList/Drawer.scss:29]: can this use spacing-small / spacing-x-small?

File: Comments/ClientApp/src/components/CommentList/Drawer.scss [hunk 3]
```diff
@@ -41,13 +38,18 @@
...
       color: $colour-nice-white;
       box-shadow: inset -10px 0 10px rgba(0, 0, 0, 0.3);
     }
-    &--comments.active,&--questions.active{
+    &--comments.active,
+    &--questions.active {
       background: $colour-btn-cta;
     }
+    &-text {
+      font-size: 0.9rem;
```
Comment (emollett) [inline Comments/ClientApp/src/components/CommentList/Drawer.scss:51]: It probably doesn't, but does this cause any issues with colour contrast? I know some of our colours only pass when at full size text.

**Review by emollett — DISMISSED** [review]

File: Comments/ClientApp/src/components/CommentList/Drawer.scss (hunk 3 above)
Comment (johnholland-nice) [reply Comments/ClientApp/src/components/CommentList/Drawer.scss:51]: Doesn't appear to be a problem when I've checked it.

**Review by emollett — APPROVED** [review]

**Review by mgallagher1980 — APPROVED** [review]

**Review by mgallagher1980 — APPROVED** [review]

**Pull request merged**